### ISSUE TITLE
pg-promise ignores specified query_timeout period 

### ISSUE THREAD

**Issue by sambidsresh** [vitaly-t/pg-promise#681]
### Expected behavior
connect() throws an error after the specified timeout period (~1000 ms).

### Actual behavior
An error is thrown an error only after default timeout period (~75000 ms).

### Steps to reproduce
Database is *not* running.
```
const pgp=require('pg-promise')();
const db=pgp({
    host: 'db host',
    port: 'db port',
    database: 'db name',
    password: 'password',
    keepAlive: 'true',
    query_timeout: 1000
});
console.time('conn_time');
db.connect()
.then(() => console.timeEnd('conn_time'))
.catch(() => console.timeEnd('conn_time'));
```

### Environment

* Version of pg-promise: 10.3.1
* OS type (Linux/Windows/Mac): Linux
* Version of Node.js: 12.6.0

**Comment by vitaly-t** [comment]
Connection timeout is determined by parameter `connect_timeout`, and not by `query_timeout`, which set timeouts for executing queries.

**Comment by sambidsresh** [comment]
I was wondering what I was wrong with connect_timeout. Thank you.

**Comment by vitaly-t** [comment]
There also has been a declaration issue, resolved in [v10.3.4](https://github.com/vitaly-t/pg-promise/releases/tag/10.3.4).

The correct way to set connection timeout is via `connectionTimeoutMillis` within the connection parameters.
